lua/nvim-paredit/init.lua

```diff
@@ -6,6 +6,8 @@ local lang = require("nvim-paredit.lang")
 
 local M = {
   api = require("nvim-paredit.api"),
+  form = require("nvim-paredit.api.form"),
```

Comment:
I don't think `form` is the best API namespace for this. Maybe `wrap`?

Reply:
don't disagree, rename incoming :)